recipes/re2/all/test_package/test_package.cpp

```diff
@@ -1,8 +1,10 @@
+#include <re2/re2.h>
+
 #include <iostream>
 #include <cassert>
-#include "re2/re2.h"
 
 int main() {
-	assert(RE2::FullMatch("hello", "h.*o"));
-	assert(!RE2::FullMatch("hello", "e"));
+    assert(RE2::FullMatch("hello", "h.*o"));
+    assert(!RE2::FullMatch("hello", "e"));
```

Comment:
assert's are optimized away in release mode.
I think we need to do this check the hard way (=using an if branch).

Reply:
yes, would be better